server/src/demo.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>SSE Example</title>
+  </head>
+
+  <body>
+    <h1>Problems leaderboard + my problems</h1>
+    <div id="sse-data"></div>
+
+    <script type="module">
+      const sseData = document.getElementById('sse-data')
+
+      const response = await fetch(
+        '/sse/sepolia/0xfa398d672936dcf428116f687244034961545d91',
+        {
+          method: 'POST',
+          headers: { 'Content-Type': 'text/event-stream' }
+        }
+      )
+      const reader = response.body
+        .pipeThrough(new TextDecoderStream())
+        .getReader()
+      while (true) {
+        const { value, done } = await reader.read()
+        if (done) break
+        sseData.innerHTML += value + '<br><br><br>'
```

Comment:
if we used `\n` or some simple JSON parsing command would it make this look prettier in the html?